Zappy_IA_Report/DTC/Code/zappyIa.py

```
@@ -0,0 +1,20 @@
+        sys.exit(84)
+
+    if len(sys.argv) == 2:
+        if sys.argv[1] == "-help":
+            info.info()
+        sys.stderr.write("Invalid number of arguments (-help for more informations).\n")
+        sys.exit(84)
+
+    args = process.iaProcess(sys.argv)
+    game.game(args)
+
+    sys.exit(0)
```

Flake8 (7.2.0)

[warning] no newline at end of file

(W292)